### ISSUE TITLE
 obligation of a precise height ?

### ISSUE THREAD

**Issue by NicolasCELLA** [MopTym/vue-waterfall#46]
Hello, it's possible to have a height  'auto'  ? Because my component is always different ! Thx

**Comment by richcoleman-alphatrust** [comment]
Yes, this component is useless in real-world/non-demo scenarios if height has to be defined.
